avocado/utils/pci.py

```diff
@@ -27,6 +27,8 @@
 from .genio import read_file
 from . import process
 
+log = logging.getLogger('avocado.test')
```

Comment:
You don't use the `log` anywhere, also the use of `avocado.test` directly from `utils` can be visible in other modules, but was declared a bad habit and new code using knowledge of `avocado` in `avocado.utils` should not be accepted...